src/tldraw/ui-overrides.ts

```diff
@@ -11,13 +14,16 @@ export function uiOverrides(plugin: TldrawPlugin): TLUiOverrides {
 	const trackEvent = useUiEvents();
 	return {
 		tools(editor, tools, helpers) {
-			// console.log(tools);
-			// // this is how you would override the kbd shortcuts
-			// tools.draw = {
-			// 	...tools.draw,
-			// 	kbd: "!q",
-			// };
-			return tools;
+			tools['lasso-select'] = {
+				id: 'lasso-select',
+				icon: 'color',
+				label: 'Lasso Select',
+				kbd: 'w',
+				onSelect: () => {
+					editor.setCurrentTool('lasso-select')
+				},
+			}
```

Comment:
Utilize the static `id` defined in the `LassoSelectTool` class
```suggestion
			tools[LassoSelectTool.id] = {
				id: LassoSelectTool.id,
				icon: 'color',
				label: 'Lasso Select',
				kbd: 'w',
				onSelect: () => {
					editor.setCurrentTool(LassoSelectTool.id)
				},
			}
```